### DISCUSSION TITLE
New base won't show up

### DISCUSSION THREAD

**Discussion by hdadhich01** [deta/cloud-docs#431] [Help]
`template` and `outbound` are just simple dictionaries
```py
catalogDB = deta.Base("catalog")
catalogDB.put(template, "template")
catalogDB.put(outbound, "outbound")
```
gives me the following
```
File "[project file path]\venv\lib\site-packages\deta\base.py", line 153, in put
    return res["processed"]["items"][0] if res and code == 207 else None
KeyError: 'processed'
```
The base doesn't show up on Deta, however, I declared a different base before this code which worked fine as shown in the following codeblock:
```py
foodDB = deta.Base("food")
foodDB.put(scrape_locations(), "locations")
foodDB.put(scrape_menus(), "menus")
```
`scrape_locations()` and `scrape_food()` both return dictionaries as well.

**Comment by jnsougata** [comment]
I think there's a problem with the `payload` you are trying to send that's why `put` can't process those items to base.
can you show us an example payload?

**Reply by hdadhich01** [reply]
Someone tracked down the error, apparently it had to do with keys being empty strings inside of the dictionary passed through `put`.

**Comment by lemonyte** [comment]
###### Already discussed on Discord, but I will post this here for easy reference.
------

Empty keys in your data, like `{"": "foo"}` will get rejected by the Base API. The `KeyError` was due to improper handling of the API's response, which should be fixed by https://github.com/deta/deta-python/pull/76. 
To make empty keys valid, the backend Base API would need to be changed, not this Python package. Depending on what kind of limitations exist, that may or may not be possible.

> Someone tracked down the error

That someone is me. It was a bit of a wild goose chase to track down the issue(s) and cause(s) 😅
